### PR TITLE
[117] Translate calendar

### PR DESCRIPTION
**Pull request by ozhurova**
Added translate calendar page

### CODE REVIEW

File: src/app/pages/calendar-page/calendar-page.component.ts [hunk 1]
```diff
@@ -59,6 +69,7 @@ export class CalendarPageComponent implements OnInit, OnDestroy {
       },
     },
     locales: [enLocale, ruLocale],
+    // locale: `${this.lang ? this.lang : 'en'}`,
```
Comment (PaulRomanov) [inline src/app/pages/calendar-page/calendar-page.component.ts:72]: remove the commented code

Comment (ozhurova) [reply src/app/pages/calendar-page/calendar-page.component.ts]: done

**Review by mhasilau — APPROVED** [review]

**Review by Nicolai-boiko — APPROVED** [review]

File: src/app/pages/calendar-page/calendar-page.component.ts [hunk 2]
```diff
@@ -48,7 +58,7 @@ export class CalendarPageComponent implements OnInit, OnDestroy {
     },
     customButtons: {
       submit: {
-        text: 'Submit',
+        text: `Submit`,
```
Comment (vstrabko) [inline src/app/pages/calendar-page/calendar-page.component.ts:61]: why those have changed?
Do we need ` here?

**Review by vstrabko — COMMENTED** [review]
Looks good!
Just a small comment from me

File: src/app/pages/calendar-page/calendar-page.component.ts (hunk 2 above)
Comment (ozhurova) [reply src/app/pages/calendar-page/calendar-page.component.ts:61]: It was changed to a dynamic value, but then returned. forgot to change

**Review by rastorc3v — APPROVED** [review]

**Pull request merged**